install_lastpass_deps.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sudo apt-get install --yes openssl libcurl3 libxml2 libssl-dev libxml2-dev libcurl4-openssl-dev pinentry-curses xclip pkg-config cmake
```

Comment:
This plus having the same packages later in salt_resources/states/lastpass/init.sls is confusing me. Where are which packages needed? Host or guest?

Reply:
you need to be able to install lastpass in the guest and in the host. that is why it is in 2 places